cmd/gpu_plugin/gpu_plugin_test.go

```diff
@@ -90,10 +90,22 @@ func TestScan(t *testing.T) {
 			},
 		},
 		{
-			name:      "one device",
+			name:      "pf-no-vfs",
+			sysfsdirs: []string{"card0/device/drm/card0", "card0/device/drm/controlD64"},
+			sysfsfiles: map[string][]byte{
+				"card0/device/vendor":       []byte("0x8086"),
+				"card0/device/sriov_numvfs": []byte("0"),
+			},
+			devfsdirs:        []string{"card0"},
+			expectedDevs:     1,
+			expectedMonitors: 1,
+		},
+		{
+			name:      "vf-no-pf",
 			sysfsdirs: []string{"card0/device/drm/card0", "card0/device/drm/controlD64"},
 			sysfsfiles: map[string][]byte{
 				"card0/device/vendor": []byte("0x8086"),
+				"card0/device/physfn": []byte("1"),
```

Comment:
Does the name "vf-no-pf" reflect the reality? IIUC it should read as "one virtual function and no physical functions", but the file "physfn" with non-zero content suggests the opposite.

Reply:
"physfn" is a (directory) symlink under VF pointing to the PF from which it's split from.

Checking for presence of this (symlink) file, is good way to check whether given device is VF.  It's contents don't matter (because even when GPU plugin would check for it's presence, it's not going to follow it), but I can use empty file content and comment about it representing symlink.

As to test name, would "one-vf-no-pf" be enough?

PS. Only documentation I found for these files is from Android kernel source tree ABI testing doc: https://android.googlesource.com/kernel/common/+/refs/heads/android-mainline/Documentation/ABI/testing/sysfs-bus-pci